src/clr-angular/utils/id-generator/id-generator.service.ts

```diff
@@ -10,7 +10,11 @@ let NB_INSTANCES = 0;
 
 export const UNIQUE_ID = new InjectionToken<string>("UNIQUE_ID");
 
+export function UniqueIdProvider() {
```

Comment:
Can we rename this `UniqueIdProvider()` function name? This function itself is not the provider. Secondly, we are exporting two different things with the same name. The only difference is one of them has uppercase letters with underscore (`UniqueIdProvider` & `UNIQUE_ID_PROVIDER`).  Maybe, `uniqueIdFactory()`?